src/plugins/intel_cpu/src/nodes/executors/acl/acl_conv.cpp

```diff
@@ -0,0 +1,149 @@
+    auto paddingTop = (attrs.paddingL.size() >= 2U) ? attrs.paddingL[0] : 0;
+    auto paddingBottom = (attrs.paddingR.size() >= 2U) ? attrs.paddingR[0] : 0;
+    padStrideInfo = arm_compute::PadStrideInfo(attrs.stride[0],
+                                               attrs.stride[1],
+                                               paddingLeft,
+                                               paddingRight,
+                                               paddingTop,
+                                               paddingBottom,
+                                               arm_compute::DimensionRoundingType::FLOOR);
+    dilation = arm_compute::Size2D(attrs.dilation[1] + 1, attrs.dilation[0] + 1);
+
+    if (attrs.postOps.size() == 1) {
+        if (const auto* const activation = std::any_cast<ActivationPostOp>(attrs.postOps.data())) {
+            activationLayerInfo = getActivationLayerInfo(convertToEltwiseAlgorithm(activation->type()),
+                                                         activation->alpha(),
+                                                         activation->beta(),
+                                                         activation->gamma());
+        } else if (const auto* const fq = std::any_cast<FakeQuantizePostOp>(attrs.postOps.data())) {
+            fqInputScale = fq->inputScale();
+            fqInputShift = fq->inputShift();
+            fqOutputScale = fq->outputScale();
+            fqOutputShift = fq->outputShift();
+            if (fqOutputScale.size() == 1 && fqOutputScale[0] == 1.0F && fqOutputShift.size() == 1 &&
+                fqOutputShift[0] == std::trunc(fqOutputShift[0])) {
+                for (auto& v : fqInputShift) {
+                    v += fqOutputShift[0];
+                }
+                fqOutputShift.clear();
+            }
+        }
+    } else {
+        DEBUG_LOG("ACLConvolutionExecutor: post op is not applied!");
+    }
+}
+
+arm_compute::Status ACLConvolutionExecutor::validateTensorsInfo(const ACLInfos& aclMemoryInfos) {
+    // quantization configuration:
+    // src scale: 1.0
+    // src shift: 0
+    // weights scale: dequantization scale fused into the conv node, or 1.0 if the scale is not defined
+    // weights shift: 0
+    // destination scale: 1.0 / FakeQuantize input scale, or 1.0 if the scale is not defined
+    // destination shift: FakeQuantize input shift, or 0 if the shift is not defined
+    aclMemoryInfos[ACLArgs::ACL_SRC_0]->set_quantization_info(arm_compute::QuantizationInfo(1.0));
+    aclMemoryInfos[ACLArgs::ACL_WEI]->set_quantization_info(
+        arm_compute::QuantizationInfo(weightScale.empty() ? 1.0F : weightScale[0]));
+    aclMemoryInfos[ACLArgs::ACL_DST]->set_quantization_info(
+        arm_compute::QuantizationInfo(fqInputScale.empty() ? 1.0F : 1.0F / fqInputScale[0],
+                                      fqInputShift.empty() ? 0 : fqInputShift[0]));
```

Comment:
Let me propose some potential improvements to the clarification. Apply if agree
```suggestion
    // Note: LPT propagate dequantization scales from src and weights on conv output, and the result scale is applied as weight scale. So quantization configuration forms in the following way:
    // - src quantization info is always trivial
    // - weights: scale is equal to result dequantization scale after Convolution propagated by LPT. Shift is not supported
    // - destination scale is formed based on requantization FakeQuantize parameters: scale = 1.0 / input scale, shift = input shift
    aclMemoryInfos[ACLArgs::ACL_SRC_0]->set_quantization_info(arm_compute::QuantizationInfo(1.0));
    aclMemoryInfos[ACLArgs::ACL_WEI]->set_quantization_info(
        arm_compute::QuantizationInfo(weightScale.empty() ? 1.0F : weightScale[0]));
    aclMemoryInfos[ACLArgs::ACL_DST]->set_quantization_info(
        arm_compute::QuantizationInfo(fqInputScale.empty() ? 1.0F : 1.0F / fqInputScale[0],
                                      fqInputShift.empty() ? 0 : fqInputShift[0]));
```

Reply:
Definitely, your comment is more meaningful then mine. Updated the comment.